index.d.ts

```diff
@@ -81,6 +81,7 @@ declare module '@primer/components' {
     children?: DetailsRenderFunction | React.ReactNode
     defaultOpen?: boolean
     overlay?: boolean
+    open?: boolean
```

Comment:
Wasn't sure if I needed to add `onToggle` here and if so what that definition would look like. It's a native prop similar to `onClick` but I don't see it in the interface for `HTMLDetailsElement`

Reply:
It doesn't look like onToggle can even be passed to a`<details />` in TypeScript?!?! 
https://codesandbox.io/s/solitary-dust-fil2m

So, `onToggle: (event: React.SyntheticEvent<HTMLDetailsElement>) => void` is a good start